web/angular-wallet/src/app/main/accounts/accounts.component.ts

```diff
@@ -90,6 +93,13 @@ export class AccountsComponent extends UnsubscribeOnDestroy implements OnInit, A
             this.storeService.getAllAccounts().then((accounts) => {
               this.accounts = accounts;
               this.dataSource.data = this.accounts;
+              
+              if (!accounts || !accounts.length) {
+                this.router.navigate(['/']);
+                this.accountService.selectAccount(null);
+              } else if (accounts.map(({ account }) => account).indexOf(this.selectedAccount.account) < 0) {
+                this.accountService.selectAccount(accounts[0]);
```

Comment:
Redirect the user if there are no accounts left. If there are accounts left, but the user deleted their currently selected account, select the first account in the list 💥

Reply:
cool